docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.8'
+services:
+  gimme_aws_creds:
+    container_name: gimme_aws_creds
+    image: gimme_aws_creds
+    build: .
+    volumes:
+      - ${HOME}/.aws/credentials:/root/.aws/credentials
+      - ${HOME}/.okta_aws_login_config:/root/.okta_aws_login_config
+    dns: # bypass dns resolution over vpn
```

Comment:
I don't feel like this should be hard-coded in the project's docker-compose file.

Reply:
@Sector95: the propose is to provide an easy and streamlined way to use gimme-aws-creds. would you please elaborate why?

Reply:
Because everyone's environment is different, and non-standard defaults tend to make diagnosing problems very difficult.

Example being, many corporate environments block DNS outbound for any IP address other than what they have configured for their network.

I can appreciate the intention of this, but ultimately default configurations should not be more elaborate than they need to be.